Core/HLE/sceKernelThread.h

```diff
@@ -26,6 +26,12 @@
 // There's a good description of the thread scheduling rules in:
 // http://code.google.com/p/jpcsp/source/browse/trunk/src/jpcsp/HLE/modules150/ThreadManForUser.java
 
+#ifdef HAVE_LIBNX
+// Conflicting types
+#define Thread _Thread
+#define ThreadContext _ThreadContext
```

Comment:
We've renamed both of these conflicts - you can remove these lines.

-[Unknown]